### ISSUE TITLE
Make gmond_service_config a class parameter

### ISSUE THREAD

**Issue by jarshwah** [jhoblitt/puppet-ganglia#61]
I'm running ganglia on some rhel 5 boxes and for some reason they expect the gmond.conf file to be in /etc/ganglia/gmond.conf rather than the /etc/gmond.conf location that params defines for rhel 5.

https://github.com/jhoblitt/puppet-ganglia/blob/master/manifests/gmond.pp#L54

Can you please consider adding this as a class parameter to gmond.pp so that I can override it?

**Comment by jarshwah** [comment]
Specifically, adding the following to ganglia::gmond class params:

```
$gmond_service_config = $::ganglia::params::gmond_service_config
```

And then..

```
file { $gmond_service_config:
```

**Comment by jhoblitt** [comment]
@jarshwah these are package from epel5?  The new param your proposing sounds reasonable.  Are you interested in taking a stab at a PR?

**Comment by jhoblitt** [comment]
It looks like the ganglia packages in epel5 were upgraded to a recent version earlier this year.  It probably makes sense to change the default paths from el5 as well.

**Comment by jarshwah** [comment]
Ah, that would explain it. I'm in the middle of a puppet 3 -> 4 migration, but if I have some time after that I'll circle back to write a PR. If the paths have just changed though it might be sufficient to simply update them in params and move on. 

From a design perspective, I favour being allowed to override everything by assigning params defaults to class parameters. It should probably be an all or nothing proposition though rather than a single parameter for a single use case. But I'll leave that decision (and implementation ;) ) to others.

**Comment by eabecho** [comment]
I'm running into this issue also.  It'd make sense to do both: update the default and add a gmond_service_config parameter.

**Comment by jhoblitt** [comment]
resolved by #67
